src/Castle.Core.Tests/project.json

```diff
@@ -2,42 +2,63 @@
   "version": "0.0.0",
   "description": "Castle.Core.Tests",
   "authors": [ "" ],
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
+  "packOptions": {
+    "tags": [ "" ],
+    "projectUrl": "",
+    "licenseUrl": ""
+  },
   "dependencies": {
-    "Castle.Core": { "target": "project", "version" : "" },
-    "Castle.Services.Logging.SerilogIntegration": { "target": "project", "version" : "" },
-    "Castle.Services.Logging.NLogIntegration": { "target": "project", "version": ""},
-    "Serilog.Sinks.TextWriter": "2.0.0-beta-505",
+    "Castle.Core": {
+      "target": "project",
+      "version": ""
+    },
+    "Castle.Services.Logging.SerilogIntegration": {
+      "target": "project",
+      "version": ""
+    },
+    "Castle.Services.Logging.NLogIntegration": {
+      "target": "project",
+      "version": ""
+    },
     "xunit": "2.1.0"
   },
-  "commands": {
-    "test": "xunit.runner.dnx"
-  },
+  "testRunner": "xunit",
   "frameworks": {
-    "dnxcore50": {
-      "compilationOptions": {
-        "define": [ "FEATURE_XUNITNET", "FEATURE_TEST_SERILOGINTEGRATION" ]
-      },
+    "netcoreapp1.0": {
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win8"
+      ],
       "dependencies": {
-        "Microsoft.CSharp": "4.0.0",
-        "System.Console": "4.0.0-beta-23516",
-        "System.Diagnostics.TraceSource": "4.0.0-beta-23516",
-        "xunit.runner.dnx": "2.1.0-rc1-build204"
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-rc2-3002702"
+        },
+        "Microsoft.CSharp": "4.0.1-rc2-24027",
```

Comment:
Shouldn't need to call these out specifically since they'd be referenced via `Microsoft.NETCore.App`